config/locales/customer_mailer.en.yml

```diff
@@ -1,14 +1,21 @@
+---
 en:
   customer_mailer:
-    welcome_email:
-      subject: Welcome to Bike Index!
+    additional_email_confirmation:
```

Comment:
Why are just the subjects in this file?

Does it make more sense to put this stuff in `en.yml`?

Reply:
> Does it make more sense to put this stuff in en.yml?

That's my inclination — I'm not sure why the mailer localization files are broken out like this. I'm following suit for now but I want to try to consolidate them into en.yml / nl.yml.

Reply:
They were put there a while ago by me, because I was just doing random things.